<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParameterTuningForm_template</class>
 <widget class="QDialog" name="ParameterTuningForm_template">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1085</width>
    <height>550</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string extracomment="Title of dialog of generation of a new timetable">Experiments for tuning a stratagy's parameters</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Experiments:</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QPushButton" name="startPushButton">
          <property name="text">
           <string>Start</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="generationSettingsPushButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Settings</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="stopSimulationAndGroupBox">
       <property name="title">
        <string>Stop experiments and...</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QPushButton" name="stopPushButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Set current best parameters</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="withoutStoppingTheSimulationGroupBox">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="title">
        <string extracomment="This is the title of a group box with more buttons, and all buttons do their function without stopping the simulation">Without stopping the experiments:</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="skipStepPushButton">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Skip current parameter combination</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="writeResultsPushButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>View current parcial report</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Progress:</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0" colspan="2">
         <widget class="Line" name="line_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="timeLabel">
          <property name="font">
           <font>
            <family>Ubuntu Mono</family>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="estTimeLabel">
          <property name="font">
           <font>
            <family>Ubuntu Mono</family>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QProgressBar" name="generationProgressBar">
          <property name="font">
           <font>
            <family>Ubuntu Mono</family>
           </font>
          </property>
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="2">
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="currentResultsTextEdit">
        <property name="font">
         <font>
          <family>Ubuntu Mono</family>
         </font>
        </property>
        <property name="lineWrapMode">
         <enum>QPlainTextEdit::WidgetWidth</enum>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="closePushButton">
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
